management-gateway/src/test/java/org/frankframework/management/bus/ResourceNotFoundExceptionTest.java

```diff
@@ -0,0 +1,31 @@
+package org.frankframework.management.bus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.logging.log4j.Level;
+import org.frankframework.core.IbisException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class ResourceNotFoundExceptionTest {
+
+	private static final String RESOURCE_NOT_FOUND_EXCEPTION_MESSAGE = "outer exception";
+
+	public static List<?> data() {
```

Comment:
I know you copied this from another test in the same package, but it is better if this method returns a `Stream<Argument>` instead.
See https://github.com/frankframework/frankframework/blob/2844695d4a4e97514dde6e618f30b3198355c8bc/cmis/src/test/java/org/frankframework/extensions/cmis/TestCreateAction.java#L42-L50 for how to write the method to build that stream.